### PR TITLE
[docs] Added development install instructions #92

### PR DESCRIPTION
**Pull request by ppabcd**
Updated README.md for Developer Environment Setup with step installation #92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,6 +170,76 @@ Now proceed with the following steps:
...
+    $ git clone https://github.com/nemesisdesign/Stouts.postfix
+```
+
+After cloning in the folder roles, go one directory up (back to the location where the roles folder is found), Create hosts file and playbook.yml with:
+```bash
+    $ touch hosts
+    $ touch playbook.yml
+```
+
+Next, write hosts file with your local machine. For example we using localhost with ip 127.0.0.1
```
Comment (nemesisdesign) [inline README.md]: from here on the text is redundant and duplicated.

Conclude the section with something like:

```
From here on you can follow the instructions available at the following sections:

- [#create-inventory-file](Create inventory file)
- [#create-playbook-file](Create playbook file)
- [#run-the-playbook](Run the playbook)

You can remove the rest of the text.
```

**Comment by ppabcd** [comment]
all done

File: README.md [hunk 2]
```diff
@@ -170,6 +170,76 @@ Now proceed with the following steps:
 Now you are ready to start configuring your network! **If you need help** you can ask questions
 on one of the official [OpenWISP Support Channels](http://openwisp.org/support.html).
 
+Install OpenWISP for Development Environment
+-------------------------------------------------
+First time create directory where you want to place OpenWISP2 configuration.
+
+For example:
+```bash
```
[the remark this answers is not in the document]
Comment (atb00ker) [reply README.md]: @ppabcd
Everytime before you start with a code block, remember to leave a line before it! (this is still pending!)

File: README.md [hunk 3]
```diff
@@ -170,6 +170,65 @@ Now proceed with the following steps:
...
+    cd ~/openwisp-dev/roles
+```
+
+Clone `ansible-openwisp2` and `Stouts.postfix` as follows:
+```bash
+    git clone https://github.com/openwisp/ansible-openwisp2.git openwisp.openwisp2
+    git clone https://github.com/nemesisdesign/Stouts.postfix
+```
+
+After cloning in the directory roles, go one directory up (back to the location where the roles directory is found), Create hosts file and playbook.yml with:
```
[the remark this answers is not in the document]
Comment (ppabcd) [reply README.md]: hmm ok i'll using `Now, go to the parent directory & create hosts file and playbook.yml:`

File: README.md [hunk 4]
```diff
@@ -278,6 +317,7 @@ create an empty file named `playbook.yml` which contains the following:
 ansible-playbook -i hosts playbook.yml -b -k -K --become-method=su
 ```
 
+
```
Comment (nemesisdesign) [inline README.md]: this new line here is not needed

File: README.md [hunk 5]
```diff
@@ -170,6 +172,43 @@ Now proceed with the following steps:
 Now you are ready to start configuring your network! **If you need help** you can ask questions
 on one of the official [OpenWISP Support Channels](http://openwisp.org/support.html).
 
+
```
Comment (nemesisdesign) [inline README.md]: this blank line is not necessary

File: README.md [hunk 6]
```diff
@@ -170,6 +172,43 @@ Now proceed with the following steps:
...
+Install OpenWISP for Development Environment
+-------------------------------------------------
```
Comment (nemesisdesign) [inline README.md]: Change to:

```
Install ansible-openwisp2 for development
-----------------------------------------
```

File: README.md [hunk 7]
```diff
@@ -170,6 +172,43 @@ Now proceed with the following steps:
...
+
+Create the directory where you want to place the repositories of the ansible roles and create directory roles.
```
Comment (nemesisdesign) [inline README.md]: Change this line to the following 3 lines:

```
If you need to modify the logic of this ansible role and you need to test your changes here we explain how to do it.

First of all, create the directory where you want to place the repositories of the ansible roles and create directory roles.
```

**Comment by ppabcd** [comment]
all done @atb00ker @nemesisdesign

**Comment by ppabcd** [comment]
please wait i'll reset last commit

**Comment by nemesisdesign** [comment]
@ppabcd I should let you wait 24 hours just so that you learn to check before pushing next time :joy: :stuck_out_tongue:

**Pull request merged**